sql.js

```diff
@@ -61,7 +61,11 @@ const applySelect = ({ select, payload, context }) => {
     const { alias, field } = part
     const key = alias || field
     if (field === '*') {
-      event[key] = json
+      if(alias) {
```

Comment:
and could you add a comment as to what causes the need for this diff treatment?

Reply:
or maybe just a link to this PR where u have the example

Reply:
Didn't catch that second one. Added a comment, let me know if it's too verbose.